### PR TITLE
Minor fixes across docs & sample-manifests

### PR DESCRIPTION
**Pull request by leo8a**

### CODE REVIEW

File: sample-manifests/spoke-mno/create_spoke.sh [hunk 1]
```diff
@@ -10,13 +10,13 @@ oc create -f 06_InfraEnv.yaml
 
 for i in {1..10}; do
 	sleep 5
-	ISO_URL=$(oc get infraenv ${CLUSTER_NAME} -o jsonpath={.status.isoDownloadURL})
-	if [[ ! -z ${ISO_URL} ]]; then
```
Comment (leo8a) [inline sample-manifests/spoke-mno/create_spoke.sh:14]: proposed changing `-n` for `! -z` as suggested by [SC2236](https://github.com/koalaman/shellcheck/wiki/SC2236), but pls check what fits u better

File: sample-manifests/spoke-sno/create_spoke_sno.sh [hunk 2]
```diff
@@ -10,13 +10,13 @@ oc create -f 06_InfraEnv.yaml
 
 for i in {1..10}; do
 	sleep 5
-	ISO_URL=$(oc get infraenv ${CLUSTER_NAME} -o jsonpath={.status.isoDownloadURL})
-	if [[ ! -z ${ISO_URL} ]]; then
```
Comment (leo8a) [inline sample-manifests/spoke-sno/create_spoke_sno.sh:14]: proposed changing `-n` for `! -z` as suggested by [SC2236](https://github.com/koalaman/shellcheck/wiki/SC2236), but pls check what fits u better

**Review by jparrill — CHANGES_REQUESTED** [review]
Hey @leo8a I've added the Namespace to every YAML associated with the RHACM because the ZTP approach from RH perspective include always RHACM and the object involved should be deployed in the proper one.

Maybe we can maintain the NS and add a Note on top of that to clarify this, WDYT?

File: sample-manifests/spoke-mno/create_spoke.sh (hunk 1 above)
Comment (jparrill) [reply sample-manifests/spoke-mno/create_spoke.sh:14]: LGTM

**Comment by leo8a** [comment]
k, makes to me. I just added the note, thanks.

**Review by jparrill — APPROVED** [review]
LGTM

**Pull request merged**